sdk-api.yaml

```diff
@@ -19,18 +19,24 @@ info:
 
 externalDocs:
   description: Stage documentation
-  url: https://docs.heystage.com
+  url: https://heystage.alphadoc.io/
 
 servers:
   - url: https://api.heystage.com/sdk-api
   - url: http://localhost:8080/sdk-api
 
 tags:
-  - name: stage # Generated class name is based on the tag; here, it's StageApi.
+  - name: Organizations
+    description: "Our customers' organizations"
+  - name: Users
+    description: "Our customers' users"
+  - name: Billing
+    description: "The checkout session for our customers"
+  - name: Stage # Generated class name is based on the tag; here, it's StageApi.
```

Comment:
We should probably remove this comment, or move it up above `tags`:

```
# Generated class name is based on the tag
tags:
  ...
```

Reply:
Sure, one sec